lua/onedarkpro/lib/compile.lua

```diff
@@ -77,6 +108,9 @@ function M.compile(theme)
             [[
 return string.dump(function()
 local set_hl = vim.api.nvim_set_hl
+local get_hl = vim.api.nvim_get_hl
```

Comment:
This will break functionality for any users < Neovim 0.9.0. Can we use `vim.api.nvim_get_hl_by_name` instead?

Reply:
I understand, but it wouldn't be possible to stick with the nvim_get_hl function. In my IDE, the function doesn't even appear to be available in my language server plugin. Perhaps it would be better to stick with nvim_get_hl instead of nvim_get_hl_by_name because it seems that the latter function will be removed in the future.

![screenshot](https://github.com/olimorris/onedarkpro.nvim/assets/50273941/e8a10803-d14f-4f08-acd3-baa4b3b88b68)

![screenshot2](https://github.com/olimorris/onedarkpro.nvim/assets/50273941/19e16968-b517-47f3-be92-70859824f8c4)

Reply:
Appreciate it will be deprecated but I don't wish to break the plugin for users who aren't on Neovim 0.9.0. In this case, I'll add a warning and ask users to pin to a "0.8.0" branch and give them two weeks notice before merging this PR